protocol/message.go

```diff
@@ -46,5 +46,16 @@ type KeyLookupRequest struct {
 type KeyLookupProof struct {
 	Type int
 	AP   *merkletree.AuthenticationPath
-	TB   *merkletree.TemporaryBinding
+	TB   *merkletree.TemporaryBinding `json:",omitempty"`
+}
+
+type ConsistencyCheckRequest struct {
+	Username string `json:"username"`
+	Epoch    uint64 `json:"epoch"`
+}
+
+type ConsistencyCheckProof struct {
```

Comment:
I'm not so sure about it. I think `ConsistencyCheckProof` shouldn't include TB field, even if the server's using TB. I'd like to have 2 separate struct to make thing more clarify. Hopefully it makes sense to you.